config/initializers/sassc_rails.rb

```diff
@@ -0,0 +1,21 @@
+require "sprockets/engines"
+
+module Extensions
+  module Sprockets
+    module Engines
+      def register_engine(ext, klass)
+        return if [
+          Sass::Rails::SassTemplate,
+          Sass::Rails::ScssTemplate
+        ].include?(klass)
+
+        super
+      end
+    end
+  end
+end
+
+Sprockets::Base.send(:prepend, Extensions::Sprockets::Engines)
```

Comment:
Should have mentioned, this is the current recommended workaround from https://github.com/bolandrm/sassc-rails/issues/6#issuecomment-100949872